erpnext_documentation/www/docs/user/manual/en/stock/purchase-return.md

```diff
@@ -37,6 +37,17 @@ Before creating and using a Purchase Return, it is advised that you create the f
 
 If Perpetual Inventory enabled, the system will also post accounting entry against warehouse account to sync warehouse account balance with stock balance as per Stock Ledger.
 
-### 3. Related Topics
+## 3. Impact on Stock Return via Purchase Receipt
+On Creating a Purchase Return against a Purchase Receipt:
+
+* The **Returned Quantity** in the original Purchase Receipt along with any Purchase Order linked to it, is updated.
+
+* The original Purchase Receipt's status is changed to **Return Issued** if 100% returned.
+  ![Return Issued](/docs/assets/img/stock/purchase-return-issue.png)
+
+* The Returned Amount % is visible via the List View as well
+  ![Purchase Receipt List View](/docs/assets/img/stock/purchase-receipt-list.png)
```

Comment:
reduce the left-hand side border (for all images)

Reply:
looks okay to me @prasadarr should i ?